quartz/components/styles/explorer.scss

```diff
@@ -1,14 +1,70 @@
 @use "../../styles/variables.scss" as *;
 
+@media all and ($mobile) {
+  .page > #quartz-body {
+    // Shift page position when toggling Explorer on mobile.
+    & > :not(.sidebar.left:has(.explorer)) {
```

Comment:
Ig we will always leave explorer to the left sidebar? If users move explorer to the right sidebar this wouldn't work

Reply:
The problem is that the right sidebar on mobile is at the very bottom of the page.

Reply:
ah i see, makes sense then